.gitignore

```
@@ -489,4 +489,4 @@ webapi/CopilotChatWebApi.sln
 /deploy/
 
 # Tesseract OCR language data files
-*.traineddata
\ No newline at end of file
+*.traineddata
```

Reverting removal of line feed

```suggestion
*.traineddata

```